README.md

```diff
@@ -24,6 +24,7 @@ typst init @preview/clean-math-paper:0.2.0
 
 ### Parameters of the Template
 
+- `lang`: Language of the paper. Supported languages are English and French, default is English.
```

Comment:
```suggestion
- `lang`: Language of the paper. Supported languages are English and French, default is "en".
```
To clarify that the language has to be provided as a language code. Otherwise one might think one has to provide "French" or so.